docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1587 @@
+{
```

Comment:
<p>This is a fantastic chart. Looks very nice! Do you think we could think about a way to make it work with plot_cap? https://github.com/bambinos/bambi/pull/517</p><p> </p><p>That implies some modifications to the predict function. I'm not 100% sure if this is possible or not. But it's worth considering it since it would make things much easier.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
I'll check this out and see if it's possible.